stable/awsebscsiprovisioner/values.yaml

```diff
@@ -1,4 +1,60 @@
+
+nameOverride: ""
+fullnameOverride: ""
+
+image:
+  repository: "amazon/aws-ebs-csi-driver"
+  tag: "v0.3.0"
+
+liveness:
+  image:
+    respository: "quay.io/k8scsi/livenessprobe"
+    tag: "v1.1.0"
+
+registrar:
+  node:
+    image:
+      repository: "quay.io/k8scsi/csi-node-driver-registrar"
+      tag: "v1.1.0"
+  cluster:
+    image:
+      repository: "quay.io/k8scsi/csi-cluster-driver-registrar"
+      tag: "v1.0.1"
+
+provisioner:
+  # True if enable volume scheduling for dynamic volume provisioning
+  enableVolumeScheduling: false
+  image:
+    repository: "quay.io/k8scsi/csi-provisioner"
+    tag: "v1.1.0"
+
+attacher:
+  image:
+    repository: "quay.io/k8scsi/csi-attacher"
+    tag: "v1.1.0"
+
+snapshotter:
+  # True if enable volume snapshot
+  enabled: false
+  image:
+    repository: "quay.io/k8scsi/csi-snapshotter"
+    tag: "v1.1.0"
+
+resizer:
+  # True if enable volume resizing
+  enabled: false
+  image:
+    repository: "quay.io/k8scsi/csi-resizer"
+    tag: "v0.2.0"
+
+# AWS key id and access key
+# these are optional and can be left as is
+secrets:
+  enabled: false
+  key_id: ""
+  access_key: ""
+
 storageclass:
-  isDefault: true
+  isDefault: false
```

Comment:
we wanted to be this true to be consistent with the other provisioners

Reply:
Seems like an arbitrary decision. Ill set it true in another PR.